src/query/select.rs

```diff
@@ -69,6 +69,11 @@ impl Select {
         self
     }
 
+    pub fn table_column_with_schema(mut self, schema: Option<&str>, table: &str, column: &str) -> Self {
```

Comment:
One adjustment I'd make: since the function explicitly is called _with_schema, I think it makes sense to take a `&str` instead of `Option<&str>`.

Likewise for the other method names.

Reply:
the problem with taking a &str is that in external calls (specifically in ormlite)..you need to have an if or a match to check for schema when creating the objects, in all cases. It's a bit...ugly.
I'm ok with either one, you pick.